lib/country_select/tag_helper.rb

```diff
@@ -13,9 +13,8 @@ def country_option_tags
         option_tags = options_for_select(priority_countries_options, option_tags_options)
         option_tags += html_safe_newline + options_for_select([priority_countries_divider], disabled: priority_countries_divider)
 
-        if priority_countries_options.map(&:second).include?(option_tags_options[:selected])
-          option_tags_options[:selected] = nil
-        end
+        option_tags_options[:selected] = [option_tags_options[:selected]] unless option_tags_options[:selected].kind_of?(Array)
```

Comment:
Right, okay, I think get that now that I realize only affects the `option` tags. Sorry about that.